### PR TITLE
Created sidenavbar and UpdateView

### PR DESCRIPTION
**Pull request by Mayankraj23**
### Description
-Created the sidenavbar according to the given color scheme and the ui.
-Used an UpdateView to update the details provided by the user but not able to implement it completely.

Fixes # [ISSUE NUMBER HERE]

### Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by maintainers)

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have commented my code or provided relevant documentation
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/accounts/models.py [hunk 1]
```diff
@@ -21,11 +22,19 @@ class TeachingAssistantProfile(models.Model):
     phone = models.CharField(max_length=10, validators=[contact])
     slug = models.SlugField(blank=True)
     teaching_assistant_supervisor = models.ForeignKey(TeachingAssistantSupervisorProfile, null=True, blank=True,
-                                                      on_delete=models.SET_NULL)
```
Comment (anshulahuja98) [inline src/accounts/models.py]: this was the correct line, you probably did wrong rebase here

File: src/accounts/models.py [hunk 2]
```diff
@@ -21,11 +22,19 @@ class TeachingAssistantProfile(models.Model):
...
+                                                      on_delete=models.DO_NOTHING)
+    alternate_pno = models.CharField(max_length=10, validators=[contact], blank=True)
+    research_area = models.CharField(max_length=100, blank=True)
+    ug_course = models.CharField(max_length=50, default= '-')
```
Comment (anshulahuja98) [inline src/accounts/models.py]: remove the default, instead add blank = True and null = True

File: src/accounts/models.py [hunk 3]
```diff
@@ -21,11 +22,19 @@ class TeachingAssistantProfile(models.Model):
...
+    pg_course = models.CharField(max_length=50, blank=True)
```
Comment (anshulahuja98) [inline src/accounts/models.py]: null = True

File: src/accounts/models.py [hunk 4]
```diff
@@ -21,11 +22,19 @@ class TeachingAssistantProfile(models.Model):
...
+
 
     def __str__(self):
         return self.roll_no + '(' + self.user.get_full_name() + ')'
 
+    def get_absolute_url(self):
```
Comment (anshulahuja98) [inline src/accounts/models.py]: ye kisliye hai?

File: src/frontend/views.py [hunk 5]
```diff
@@ -8,8 +8,11 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
 from courses.models import Course, Feedback
 from .forms import ApprovalForm
+
```
Comment (anshulahuja98) [inline src/frontend/views.py]: remove all these blank lines, auto indent code in pycharm with Ctrl+alt+L

File: src/accounts/models.py (hunk 4 above)
Comment (Mayankraj23) [reply src/accounts/models.py]: After clicking on the update to redirect the url to the details page..ye nahi lgaega toh error aega

**Pull request closed without merging**